### DISCUSSION TITLE
Welcome to alpha-beta-pruning-minmax-checkers Discussions!

### DISCUSSION THREAD

**Discussion by njmarko** [njmarko/alpha-beta-pruning-minmax-checkers#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nikJ13** [comment]
Hey @njmarko ! Have you used sequential alpha beta pruning or parallel alpha beta pruning?

**Reply by njmarko** [reply]
Hi @nikJ13 .  I used sequential alpha beta pruning here.
